mysql-test/main/func_misc.test

```diff
@@ -1278,3 +1278,59 @@ DROP TABLE t1;
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
+--echo #
+--echo # MDEV-4017 - GET_LOCK() with negative timeouts has strange behavior
+--echo #
+
+--echo #
+--echo # MDEV-10569 Add RELEASE_ALL_LOCKS SQL-function 
+--echo # 
+
+--echo # Test function without any locks
+SELECT RELEASE_ALL_LOCKS();
+
+--echo # Test function with one lock only
+SELECT GET_LOCK ('l1',10);
+SELECT RELEASE_ALL_LOCKS();
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info;
+                                                         
+--echo # Test function with multiple locks
+SELECT GET_LOCK ('l1',10),
+       GET_LOCK ('l2',10),
+       GET_LOCK ('l3',10),
+       GET_LOCK ('l4',10),
+       GET_LOCK ('l5',10),
+       GET_LOCK ('l6',10),
+       GET_LOCK ('l7',10),
+       GET_LOCK ('l8',10),
+       GET_LOCK ('l9',10),
+       GET_LOCK ('l10',10),
+       GET_LOCK ('l11',10),
+       GET_LOCK ('l12',10),
+       GET_LOCK ('l13',10),
+       GET_LOCK ('l14',10),
+       GET_LOCK ('l15',10);  
+SELECT LOCK_MODE, LOCK_TYPE, TABLE_SCHEMA, TABLE_NAME FROM information_schema.metadata_lock_info;
```

Comment:
Order of records is undefined here and may cause sporadic failures. Add `ORDER BY TABLE_NAME`.